server/routes/ivory-age.route.js

```diff
@@ -0,0 +1,125 @@
+'use strict'
+
+const { Paths, RedisKeys, Views } = require('../utils/constants')
+const RedisService = require('../services/redis.service')
+const { buildErrorSummary, Validators } = require('../utils/validation')
+
+const itemType = 'musical' // temporary to deal with dynamic heading until parent page built. Values: 'musical', '10%' or 'other'
+const highValue = false // temporary to deal with dynamic page content until parent page built
+
+const handlers = {
+  get: async (request, h) => {
+    return h.view(Views.IVORY_AGE, {
+      ..._getContext()
+    })
+  },
+  post: (request, h) => {
```

Comment:
Trivial issue but I prefer having a blank line before line 16  as it makes the get and post easier to distinguish when scanning quickly though the code